### ISSUE TITLE
Adding the "Originator" ...

### ISSUE THREAD

**Issue by t117tim** [jasonjoh/send-actionable-message#2]
Hi Jason,
This CODE is AWESOME ! ! !

One tip that would make it even better.

You should advise on the readme/front page, that if you want to use it to send email to another person (to really prove the card is working - and not just test with yourself), ... you need to:
1. register as a developer (https://outlook.office.com/connectors/oam/publish)
2. put the provider ID into the config file, (Just like you do with the application id).

It took me a while to work this out and I went in a few a circles trying to get it to work.
It is well described here: https://docs.microsoft.com/en-us/outlook/actionable-messages/send-via-email
![ActionableDeveloperDashboard](https://user-images.githubusercontent.com/46833712/74894995-95da9300-53e4-11ea-9523-61a09d8818d9.png)
![ConfigFile](https://user-images.githubusercontent.com/46833712/74894997-970bc000-53e4-11ea-9e1f-c27b092e776f.png)
![OriginatorField](https://user-images.githubusercontent.com/46833712/74894999-983ced00-53e4-11ea-91ae-fee0d47e2e42.png)

But nice - really nice !

**Comment by IndigoShock** [comment]
> Hi Jason,
> This CODE is AWESOME ! ! !
> 
> One tip that would make it even better.
> 
> You should advise on the readme/front page, that if you want to use it to send email to another person (to really prove the card is working - and not just test with yourself), ... you need to:
> 
> 1. register as a developer (https://outlook.office.com/connectors/oam/publish)
> 2. put the provider ID into the config file, (Just like you do with the application id).
> 
> It took me a while to work this out and I went in a few a circles trying to get it to work.
> It is well described here: https://docs.microsoft.com/en-us/outlook/actionable-messages/send-via-email
> ![ActionableDeveloperDashboard](https://user-images.githubusercontent.com/46833712/74894995-95da9300-53e4-11ea-9523-61a09d8818d9.png)
> ![ConfigFile](https://user-images.githubusercontent.com/46833712/74894997-970bc000-53e4-11ea-9e1f-c27b092e776f.png)
> ![OriginatorField](https://user-images.githubusercontent.com/46833712/74894999-983ced00-53e4-11ea-91ae-fee0d47e2e42.png)
> 
> But nice - really nice !

Agreed, I had to go through the same process as you. I was checking for why I wasn't able to communicate between boxes but once I added in a valid originatorId (and in Program.cs, turned off the IsValidEmail function and hardcoded in a valid email), then it now works perfectly
